string first_name
string last_name
string patronymic
---
string full_name
